### PR TITLE
Pillars Section

### PR DESCRIPTION
**Pull request by dh-i**
@tridenda created pillars sections and fixed hero section font sizing, review and merge.

### CODE REVIEW

File: src/components/sections/hero/hero.component.tsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import Image from "next/image";
-import heroImg from "public/static/hero-logo-img.png";
+import heroImg from "public/images/hero-logo-img.png";
```
Comment (triwritescode) [inline src/components/sections/hero/hero.component.tsx:2]: I found module errors while I'm running Storybook, please check the Storybook view to check and refine them.

File: src/components/sections/pillars/icons/index.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import Image from "next/image";
+import family from "public/images/icons/pillars/family.svg";
+import health from "public/images/icons/pillars/health.svg";
+import lightBulb from "public/images/icons/pillars/lightbulb.svg";
```
Comment (triwritescode) [inline src/components/sections/pillars/icons/index.tsx:4]: please check this too

Comment (dh-i) [reply src/components/sections/pillars/icons/index.tsx:4]: @tridenda you need to be specific. What's there to check? 

I guess you are not using the workflow in the right way.

File: src/components/sections/hero/hero.component.tsx (hunk 1 above)
Comment (dh-i) [reply src/components/sections/hero/hero.component.tsx:2]: @tridenda You need to be more specific about the error. Can you share the screenshot of the same and also make sure you are using the new snapshot which has storybook-addon-next installed. 

or May be you have improperly merged the code.

**Comment by dh-i** [comment]
https://user-images.githubusercontent.com/51533230/235877397-35c300d8-8cf9-45de-b43f-fd1b25f45d3a.mov

**Pull request merged**